tests/Application/.env

```diff
@@ -12,12 +12,9 @@ APP_SECRET=EDITME
 # Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # For a sqlite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
 # Set "serverVersion" to your server version to avoid edge-case exceptions and extra database calls
-DATABASE_URL=mysql://root@127.0.0.1/coop_tilleuls_sylius_click_n_collect_plugin_%kernel.environment%?serverVersion=5.5
+DATABASE_URL=mysql://root:mysql@127.0.0.1/coop_tilleuls_sylius_click_n_collect_plugin_%kernel.environment%?serverVersion=8.0
 ###< doctrine/doctrine-bundle ###
 
-###> symfony/swiftmailer-bundle ###
-# For Gmail as a transport, use: "gmail://username:password@localhost"
-# For a generic SMTP server, use: "smtp://localhost:25?encryption=&auth_mode="
-# Delivery is disabled by default via "null://localhost"
-MAILER_URL=smtp://localhost
-###< symfony/swiftmailer-bundle ###
+MAILER_DSN=nul://null
```

Comment:
Flex markup is missing, shouldn't this variables be added/removed by Flex?

Reply:
Flex was installed globally in the CI, should I add it and run recipes ?